docs/azure/ContinuousIntegration.rst

```diff
@@ -151,31 +151,92 @@ Create a secondary tenant (Azure Active Directory B2C)
 
 #. Link this Azure AD B2C tenant to the subscription for CI by following the `Billing guide <https://docs.microsoft.com/en-us/azure/active-directory-b2c/billing#link-an-azure-ad-b2c-tenant-to-a-subscription>`_.
 
```

Comment:
As a lot of stuff in this piece of documentation is similar with the Continuous Deployment, make the same changes where applicable.